### ISSUE TITLE
Main roads & railroads hard to find

### ISSUE THREAD

**Issue by thany** [mapsme/omim#6100]
At any zoom level, main roads (like highways, freeways, motorways, expressways, etc) are hard to distinguish from other, secondary roads. They're all orange.

Because of this, and the mish-mash of green and yellow for empty land (suppose I can call it that) in certain places, makes the map look extremely cluttered and makes it hard to find anything a medium zoom levels.

This problem goes away at high zoom levels (meaning zoomed in), but only in the cities. In the outskirts, the green-yellow mess is just magnified. At low zoom levels (meaning zoomed out), the map looks far less messy but also lacks almost all meaningful detail, like [placenames and important station names](https://github.com/mapsme/omim/issues/6099).

To recap:
1) Please add better distinction between types of roads.
2) Please make the green-yellow mess less messy, by removing the green slabs where they only add clutter. Things like city parks should (of course!) still show green.
3) Perhaps add options to show or hide nonessential details.

**Comment by burivuh** [comment]
Hi!

Have redirected this to our map designer. He will look into this.

Thank you!
